templates/tfengine/recipes/cicd.hcl

```diff
@@ -259,6 +259,32 @@ schema = {
               }
             }
           }
+          worker_pool = {
+            description          = <<EOF
+              Optional worker pool attribute. Required for CICD to work with private cluster endpoints.
```

Comment:
Make the comment a bit more general. For example:

Optional Cloud Build private worker pool configuration. Required for CICD to access resources in a private network, .e.g GKE clusters with a private endpoint.

Reply:
done